openedx/adg/lms/courseware_override/tests/test_views.py

```diff
@@ -0,0 +1,123 @@
+"""
+Tests for views.py file of courseware_override
+"""
+import mock
+import pytest
+
+from common.djangoapps.student.roles import CourseInstructorRole
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.adg.lms.applications.tests.factories import MultilingualCourseFactory, MultilingualCourseGroupFactory
+from openedx.adg.lms.courseware_override.views import (
+    get_all_course_instructors,
+    get_all_languages_for_course,
+    get_course_enrollment_count
+)
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+
+from .constants import DUMMY_IMAGE_URLS, END_DATE, START_DATE
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='users')
+def user_fixture(request):
+    """
+    Create test users
+
+    Returns:
+        list: A list of User objects as specified
+    """
+    users = UserFactory.create_batch(request.param)
+    return users
+
+
+@pytest.fixture(scope='function', name='get_course')
+def course_fixture():
+    """
+    Creates a function to create a course with the given language
+
+    Returns:
+        func: A function that allows user to create a course with the specified language
+    """
+    def create_course(language='en'):
```

Comment:
We can do ```def create_course(**kwargs)``` and then we will have a generic course overview fixture.

Reply:
I actually only needed language for the current tests but yes, we could definitely create one.